scripts/screeps_rl_training_runner.py

```
@@ -1048,9 +1058,352 @@ def build_training_report(
+
+def aggregate_policy_reward_tuple(summaries: Sequence[JsonObject]) -> list[float | int]:
+    tuples: list[list[float]] = []
+    for summary in summaries:
+        reward = summary.get("reward")
+        raw_tuple = reward.get("tuple") if isinstance(reward, dict) else None
+        if not isinstance(raw_tuple, list) or len(raw_tuple) < len(REWARD_TIERS):
+            continue
+        tuples.append([float(value) for value in raw_tuple[: len(REWARD_TIERS)]])
+    if not tuples:
+        return [0 for _tier in REWARD_TIERS]
+    columns = zip(*tuples)
```

Ruff (0.15.12)

[warning] `zip()` without an explicit `strict=` parameter

Add explicit value for parameter `strict=`

(B905)